sys/net/gnrc/netif/gnrc_netif_device_type.c

```diff
@@ -54,6 +54,81 @@ netopt_t gnrc_netif_get_l2addr_opt(const gnrc_netif_t *netif)
     return res;
 }
 
+#if defined(MODULE_CC110X) || defined(MODULE_NRFMIN)
+static void _create_eui64_from_short(const uint8_t *addr, size_t addr_len,
+                                     eui64_t *eui64)
+{
+    const unsigned offset = sizeof(eui64_t) - addr_len;
+
+    memset(eui64->uint8, 0, sizeof(eui64->uint8));
```

Comment:
why did you not use an assignment statement here?
```c
eui64->uint8[0] = 0x00;
```

Reply:
okay, further down I did see that you basically moved this function up without any modifications.

Reply:
because `eui64->uint8` is an array, so

```C
memset(eui64->uint8, 0, sizeof(eui64->uint8)); /* sets all elements of the array to 0 */
```

is not the same 

as 

```C
eui64->uint8[0] = 0x00; /* sets the first element of the array to 0 */
```

;-P

Reply:
mhm, true (: never mind then